modules/jaeger-install-overview.adoc

```diff
@@ -1,7 +1,8 @@
 ////
-This CONCEPT module included in the following assemblies:
+This module included in the following assemblies:
 - rhbjaeger-installation.adoc
 ////
+:pantheon-module-type: CONCEPT
```

Comment:
The content type for the file. Replace <TYPE> with the actual type of the module, CONCEPT, REFERENCE, or PROCEDURE. Place this attribute before the anchor ID or, if present, the conditional that contains the anchor ID.

Refer to the guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Follow the same style.